app/helpers/curate_helper.rb

```diff
@@ -182,11 +177,24 @@ def sorted_college_list_for_generic_works
       list[k]["label"]
     end.sort << "Other"
   end
-
+    def sorted_college_list_for_degrees
```

Comment:
I think you should keep the old method name and just replace the contents of the method with your new code. And get rid of your *_new method. Does that make sense?